Fixes error
./stdio.texi:3269: First argument to cross-reference may not be empty.
./stdio.texi:3270: First argument to cross-reference may not be empty.
makeinfo: Removing output file `/home/dank/downloads/crosstool-0.24/build/mipsel-unknown-linux-gnu/gcc-3.2.3-glibc-2.2.3/glibc-2.2.3/manual/libc.info' due to errors; use --force to preserve.
make[2]: *** [libc.info] Error 2
make[2]: Leaving directory `/home/dank/downloads/crosstool-0.24/build/mipsel-unknown-linux-gnu/gcc-3.2.3-glibc-2.2.3/glibc-2.2.3/manual'

===================================================================
RCS file: /cvs/glibc/libc/manual/stdio.texi,v
retrieving revision 1.126
retrieving revision 1.127
diff -u -r1.126 -r1.127
--- libc/manual/stdio.texi	2001/06/06 07:11:31	1.126
+++ libc/manual/stdio.texi	2001/07/31 18:57:16	1.127
@@ -3265,8 +3265,8 @@
 If you are trying to read input that doesn't match a single, fixed
 pattern, you may be better off using a tool such as Flex to generate a
 lexical scanner, or Bison to generate a parser, rather than using
-@code{scanf}.  For more information about these tools, see @ref{, , ,
-flex.info, Flex: The Lexical Scanner Generator}, and @ref{, , ,
+@code{scanf}.  For more information about these tools, see @ref{Top, , ,
+flex.info, Flex: The Lexical Scanner Generator}, and @ref{Top, , ,
 bison.info, The Bison Reference Manual}.
 
 @node Input Conversion Syntax
